bentoml/_internal/runner/runner.py

```diff
@@ -105,7 +105,7 @@ def required_models(self) -> t.List[Tag]:
         return []
 
     @abstractmethod
-    def _setup(self, **kwargs: t.Any) -> None:
+    def _setup(self) -> None:
```

Comment:
@bojiang can you confirm this won't break anything? Test does pass. The reason behind this is that `_setup()` is not user facing, so anything a custom runner needs to pass into setup can be passed via `__init__`

Reply:
I think this is correct, setup should not take additional arguments